app/uk/gov/hmrc/servicedependencies/DataReloadScheduler.scala

```diff
@@ -86,7 +70,7 @@ object MicroserviceGlobal extends GlobalSettings
       Logger.warn(s"$libraryReloadIntervalKey is missing. reload will be disabled")
     } { reloadInterval =>
       Logger.warn(s"libraryReloadIntervalKey set to $reloadInterval minutes")
-      val cancellable = UpdateScheduler.startUpdatingLibraryData(reloadInterval minutes)
+      val cancellable = updateScheduler.startUpdatingLibraryData(reloadInterval minutes)
       app.injector.instanceOf[ApplicationLifecycle].addStopHook(() => Future(cancellable.cancel()))
```

Comment:
Here as well ApplicationLifecycle can be injected directly

Reply:
👍